opslevel/resource_opslevel_check_package_version.go

```diff
@@ -170,10 +170,10 @@ func (r *CheckPackageVersionResource) ValidateConfig(ctx context.Context, req re
 			}
 		}
 	} else {
-		if !configModel.MissingPackageResult.IsNull() {
+		if !configModel.MissingPackageResult.IsUnknown() && !configModel.MissingPackageResult.IsNull() {
```

Comment:
Do we need to changelog this bug fix?

Reply:
Yes! I knew this needed to be done then forgot it 😅 - adding this in